WrocRide/Services/AccountService.cs

```diff
@@ -120,6 +120,7 @@ public void RegisterDriver(RegisterDriverDto dto)
                 };
 
                 _dbContext.Documents.Add(document);
+                _dbContext.SaveChanges();
```

Comment:
To jest niepotrzebne bo po dodaniu elementu car robimy zapis zmian. Ten zapis sie odbywa przed utworzeniem tabeli w tabeli drivera